lib/link.js

```diff
@@ -50,6 +50,14 @@ function link (args, cb) {
   linkPkg(npm.prefix, cb)
 }
 
+function parentFolder (id, folder) {
```

Comment:
You have more faith in software to not do the most perverse possible thing than I do, but I defer to your judgment.